script.js

```diff
@@ -0,0 +1,81 @@
+const container = document.querySelector('.mobile-header-nav-links');
+const icon = document.querySelector('.menu');
+const times = document.querySelector('.close');
+
+function open() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.